### ISSUE TITLE
Consider GitHub Actions variables

### ISSUE THREAD

**Issue by lread** [liquidz/antq#70]
When I run antq on rewrite-clj, I get:
```
|                                   :file |                     :name |            :current |     :latest |
|-----------------------------------------+---------------------------+---------------------+-------------|
| .github/workflows/native-image-test.yml | graalvm/graalvm-ce-builds | ${{ matrix.graal }} | vm-21.0.0.2 |
```
My GitHub Actions workflow has:
```yaml
    - name: Install GraalVM
      uses: DeLaGuardo/setup-graalvm@master
      with:
        graalvm-version: ${{ matrix.graal }}
```
I suppose antq could figure out possible variables for `${{ matrix.graal }}`:
```yaml
    strategy:
      matrix:
        os: [ windows-latest, ubuntu-latest, macos-latest ]
        graal: [ 21.0.0.2.java8, 21.0.0.2.java11 ]
        test: [ pure, sci ]
        exclude:
          - os: windows-latest
            graal: 21.0.0.2.java8
```
But perhaps easier to just ignore this case?

**Comment by liquidz** [comment]
@lread Thanks for your reporting!
I'll consider how to deal with it.

**Comment by liquidz** [comment]
@lread I tried to support `matrix` in [dev](https://github.com/liquidz/antq/tree/dev) branch.

Could you try?
- checkout `dev` branch
- run `make uberjar` to generate target/antq-standalone.jar
- run `java -jar /path/to/your/antq/repo/target/antq-standalone.jar` in your project directory

**Comment by lread** [comment]
@liquidz just tried it out!  

I just noticed I am not following the latest conventions for the setup-graalvm action, I use:
```yaml
      uses: DeLaGuardo/setup-graalvm@master
      with:
        graalvm-version: ${{ matrix.graal }}
```

Whereas the current docs show:
```yaml
- uses: DeLaGuardo/setup-graalvm@4.0
  with:
    # GraalVM version, no pattern syntax available atm
    graalvm: '21.0.0.2'
    # Java version, optional, defaults to 'java8'. Available options are 'java8' and 'java11'.
    java: 'java11'
    # Architecture flag, optional, defaults to 'amd64'. Available options are 'amd64' and 'aarch64'. Later is available only for linux runners.
    arch: 'amd64'
```

So the fact that antq even found my graalvm version is pretty impressive.

If I downgrade my current matrix.graal to `[ 21.0.0.1.java8, 21.0.0.1.java11 ]`, I get:

```Shell
|                                   :file |                     :name | :current |     :latest |
|-----------------------------------------+---------------------------+----------+-------------|
| .github/workflows/native-image-test.yml | graalvm/graalvm-ce-builds | 21.0.0.1 | vm-21.0.0.2 |
```

This helpfully tells me I could be using a later version of graal!  
Curious, what does the `vm-` prefix mean?

If I switch to the newer recommended setup-graalvm syntax, I don't really have a matrix thing going on for the graal version anymore as I always want to use the latest, but let's try it.

My updated matrix:
```yaml
    strategy:
      matrix:
        os: [ windows-latest, ubuntu-latest, macos-latest ]
        graal-java: [ java8, java11 ]
        test: [ pure, sci ]
        exclude:
          - os: windows-latest
            graal-java: java8
```

And my setup-graalvm step (with an older version of graalvm specified):
```yaml
    - name: Install GraalVM
      uses: DeLaGuardo/setup-graalvm@4.0
      with:
        graalvm: '21.0.0.1'
        java: ${{ matrix.graal-java }}
```

In this case antq does not report that my graalvm version is out of date.

**Comment by liquidz** [comment]
@lread Thanks!

> I just noticed I am not following the latest conventions for the setup-graalvm action
> ...
> In this case antq does not report that my graalvm version is out of date.

Oh, thanks for your notification!
I fixed it in `dev` branch (5921a56)

> Curious, what does the vm- prefix mean?

[graalvm/graalvm-ce-builds](https://github.com/graalvm/graalvm-ce-builds/tags) has `vm-` prefix in tags.
antq compares the version number and latest tag name, so it displays as such for now.

**Comment by lread** [comment]
Thanks @liquidz, your fix on dev looks good from here!

**Comment by liquidz** [comment]
@lread Thanks for your confirmation!
I just released 0.12.4
